Xamarin.Forms.Platform.MacOS/Renderers/ScrollViewRenderer.cs

```diff
@@ -19,10 +19,10 @@ public class ScrollViewRenderer : NSScrollView, IVisualElementRenderer
 
 		public ScrollViewRenderer() : base(RectangleF.Empty)
 		{
+            ContentView = new FlippedClipView();
 			DrawsBackground = false;
-			ContentView.PostsBoundsChangedNotifications = true;
-			NSNotificationCenter.DefaultCenter.AddObserver(this, new Selector(nameof(UpdateScrollPosition)),
-				BoundsChangedNotification, ContentView);
+            ContentView.PostsBoundsChangedNotifications = false;
+			NSNotificationCenter.DefaultCenter.AddObserver(this, new Selector(nameof(UpdateScrollPosition)), BoundsChangedNotification, ContentView);
```

Comment:
That is so weird.  When viewed in my VS2015 instance it shows the correct number if tabs.  I'll remove them and tab them in.  Should be up soon.  Thanks.